extensions/azurecompute.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: AzureCompute
+args:
+  instance_name:
+    type: string
+    required: true
+  region:
+    type: string
+    required: true
+    extension_arg: region
```

Comment:
Just fyi; if the arg name matches the key name (L7) you don't need to supply this. Nothing we need to block on.